sql/database.sql

```diff
@@ -1485,6 +1485,8 @@ CREATE TABLE `drugs` (
   `drug_code` varchar(25) NULL,
   `consumable` tinyint(1) NOT NULL DEFAULT 0 COMMENT '1 = will not show on the fee sheet',
```

Comment:
upgrade scripts adds 17 last_updated/date_created items, but only 15 in the database.sql. So looks like database.sql is missing 2 :)

Reply:
Good catch! Haha!  Got it in.